test/e2e/specs/fakerp/loglevel.go

```diff
@@ -0,0 +1,60 @@
+package fakerp
+
+import (
+	"context"
+	"net/http"
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/Azure/go-autorest/autorest/to"
+
+	"github.com/openshift/openshift-azure/test/clients/azure"
+	"github.com/openshift/openshift-azure/test/e2e/standard"
+)
+
+var _ = Describe("Change OpenShift Component Log Level E2E tests [ChangeLogLevel][Fake][LongRunning]", func() {
+	var (
+		azurecli *azure.Client
+		cli      *standard.SanityChecker
+		ctx      = context.Background()
+	)
+
+	BeforeEach(func() {
+		var err error
+		azurecli, err = azure.NewClientFromEnvironment(true)
+		Expect(err).NotTo(HaveOccurred())
```

Comment:
add `Expect(azurecli).NotTo(BeNil())`

Reply:
Our tests grow a log when we add lines like these - I am happy to, and would rather, code on the assumption that if no error is returned, the return value is non-nil.